src/components/modals/StatsModal.tsx

```diff
@@ -142,6 +144,19 @@ export const StatsModal = ({
           <MigrationIntro handleMigrateStatsButton={handleMigrateStatsButton} />
         </div>
       )}
+      <hr className="mt-4 -mb-4 border-gray-500" />
```

Comment:
You'll want to wrap this with a boolean like on line 92:

`{(isGameLost || isGameWon) && ( your code ) : <></>`

Reply:
The hr I have there is breaking that block for some reason. any ideas?

Reply:
yeah, the problem is you can only have one child element inside a conditional. The solution is to wrap it all in a single tag.